### PR TITLE
feat(index): update reference to platform SDK docs for 1.13

### PR DESCRIPTION
**Pull request by morphis**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Along with bug fixes and general improvements, Anbox Cloud 1.13 comes with:
 | 2 | ref/ams-http-api | [AMS HTTP API](https://discourse.ubuntu.com/t/ams-rest-api-reference/17801)|
 | 2 | ref/anbox-https-api | [Anbox HTTP API](https://discourse.ubuntu.com/t/anbox-http-api-reference/17819)|
 | 2 | ref/anbox-stream-gateway | [Stream Gateway API](https://anbox-cloud.github.io/1.10/anbox-stream-gateway/index.html)|
```
Comment (ru-fu) [inline index.md:120]: Do we have a new version of this one as well?

**Review by ru-fu — CHANGES_REQUESTED** [review]
This also needs an update in reference/sdks.md.

**Comment by ru-fu** [comment]
There's also links in reference/anbox-http-api.md and reference/platforms.md .

File: index.md (hunk 1 above)
Comment (morphis) [reply index.md:120]: Not yet, the docs are currently not building and we have to fix that first.

**Review by ru-fu — APPROVED** [review]

**Pull request merged**